docs/README.md

```diff
@@ -25,6 +25,10 @@ Check out the [Showcase](https://github.com/docsifyjs/awesome-docsify#showcase)
 
 Please consider donating if you think docsify is helpful to you or that my work is valuable. I am happy if you can help me [buy a cup of coffee](https://github.com/QingWei-Li/donate). :heart:
```

Comment:
Should we update to OpenCollective either？ since it is going to be removed from the badges in this changes either.

Reply:
Keep it. Buy him a cup of coffee.

Reply:
👌